src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,181 @@
+
+- minutes: 5, 10, 30, 60
+- hours: 3, 12, 24
+- one week or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+[{% asset notifications/alert_frequency.png alt="A dropdown that adjusts the frequency of an alert." %}]({% asset notifications/alert_frequency.png @path %})
+
+### Environment
+Environment control allows you to specify which environment qualifies for your issue alert.
+
+- All Environments
+- Any one of your defined (and not hidden) environments
+
+[{% asset notifications/alert_environment.png alt="A dropdown allowing you to choose an environment to attach to an alert." %}]({% asset notifications/alert_environment.png @path %})
+
+If you select "All Environments", Sentry checks individually for each environment, rather than combined events across environments.
+
+### Digests
+
+The digests feature works only for **issue alert emails** and limits alerts across projects. This project-level setting allows you to batch issue alerts to limit the total number of emails you receive for that project. Use the sliders to control the frequency.
+
+[{% asset notifications/alert_digest.png alt="A sliding adjustment scale for the frequency of alert emails." %}]({% asset notifications/alert_digest.png @path %})
+
+## Alert Listing
+
+This is a listing of all your project's alerts.
+
+[{% asset notifications/alert-listing.png alt="Alert rules are divided by projects, their conditions/triggers, and actions(s)." %}]({% asset notifications/alert-listing.png @path %})
+
+## Alert Subscription
+
+### Issue Owners
+
+By default, [LINK: issue owners] receive alert emails. If an issue owner is not configured or not found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+
+[{% asset notifications/issue_owners2.png alt="A toggle indicating if all users are issue owners or not." %}]({% asset notifications/issue_owners2.png @path %})
+
+### Ignore
+
+When an issue is set to ignore, Sentry mutes the alerts for that issue. You can also ignore an issue until certain conditions are met —- for example, "ignore for 30 minutes." Keep in mind; **an ignored issue will still count towards your quota**.
+
+### Personal Alert Settings
+
+Review your personal alert settings in **User Settings > Account > Notifications**.
+
+Use the "default project alerts" setting to set your default preference across all projects — subscribed or unsubscribed.
+
+[{% asset notifications/default_project_alerts.png alt="A toggle for turning on or off all project alerts." %}]({% asset notifications/default_project_alerts.png @path %})
+
+After selecting the appropriate alert setting, selectively change it by project in **User Settings > Account > Fine tune alerts by project**.
+
+Each project has three options: Default, On, or Off. Selecting default uses your default preference from the previous step.
```

Comment:
Note that metric alerts has an action to overwrite your personal alert settings. Leaving it up to you as to whether you want to mention this.  

Additional FYI, afaik, metric alerts and issue alerts don't have the same actions.

Reply:
@adhiraj I feel like we should document how Metric Alerts has an action to overwrite personal alert settings. Will you provide the content for docs or give me some more details?

Reply:
Sure, let's put it after "...subscribed or unsubscribed."?

suggestion: Note that this setting does not affect alerts configured to be sent specifically to your email.